tests/test_metadata_utils.py

```diff
@@ -50,6 +50,13 @@ def setUp(self) -> None:
                     {"key": "html", "type": "local", "value": "i", "char_start_idx": 43, "char_end_idx": 48},
                 ],
             },
+            {
+                "id": "0003",
+                "text": "Wubba Lubba Dub Dub!",
+                "metadata": [
+                    {"key": "url", "type": "global", "value": "callto:RickAndMorty/Year%202021/"},
```

Comment:
`callto:` isn't really necessary, just want to ensure the implementation isn't vulnerable to rare URI schemes.

Reply:
Sure, makes sense to check for such corner cases 👍